lite/proxy/proxy.go

```diff
@@ -9,7 +9,7 @@ import (
 	rpcclient "github.com/tendermint/tendermint/rpc/client"
```

Comment:
Still need CHANGELOG_PENDING entry, should be under v0.26.2 now. This is a BREAKING CHANGE to the GO API, most especially because these functions are now blocking. We need to be very explicit about this change in behaviour since in-process users of Tendermint will likely have to update their code to successfully upgrade, else they'll be left scratching their heads when their nodes don't start properly.